src/library_glut.js

```diff
@@ -310,22 +336,27 @@ var LibraryGLUT = {
     GLUT.initTime = Date.now();
 
     var isTouchDevice = 'ontouchstart' in document.documentElement;
+    if (isTouchDevice) {
+      // Historically handling of touch events was made in completely wrong way.
```

Comment:
I find these kind of comments to be a bit bad, since they describe the process of how the code got to the current state and navigated through the bugs it had, but it would be better to focus on documenting the current state instead.  Having comments that mention that "there used to be a bug here, but it's now fixed" have no value - commit messages are good for carrying that kind of information instead.